stable/insights-agent/templates/kubesec/serviceaccount.yaml

```diff
@@ -0,0 +1,8 @@
+{{- if .Values.kubesec.enabled -}}
```

Comment:
I'd say let's not bother with an individual service account when there's no RBAC, but I'd be happy to hear other thoughts on this.

Reply:
I think we definitely need some RBAC for kubesec, it `kubectl get`s a bunch of controllers.

Reply:
`kubesec` uses/gets/list these resources `deployments,statefulsets,daemonsets,namespaces`